tools/distribution/src/release/assets/gh_asset.py

```diff
@@ -167,7 +167,7 @@ def validate(self, zip_directory: DirectoryMatcher, in_version: Version) -> bool
         if in_version < min_cr_version:
             return False # Datadog Crash Reporting.xcframework was introduced in `1.7.0`
 
-        dir = zip_directory.get('CrashReporter.xcframework')
+        dir = zip_directory.get(self.name)
```

Comment:
Is there a reason for not doing this change in other `validate` functions?

Reply:
That's a miss, let me update all of them 👍